synapse/lib/cell.py

```diff
@@ -876,24 +950,10 @@ def addHttpApi(self, path, ctor, info):
         ))
 
     async def _initCellDmon(self):
-        # start a unix local socket daemon listener
-        sockpath = os.path.join(self.dirn, 'sock')
-        sockurl = f'unix://{sockpath}'
-
-        self.dmon = await s_daemon.Daemon.anit()
+        certdir = s_common.gendir(self.dirn, 'certs')
```

Comment:
This is a backwards incompatible change that affects people who are currently using TLS

Reply:
I guess i can undo it, but we'll be back to having implicit shit in the running user's home directory rather than self-contained devops-able deployments...